docs/quickstart.ipynb

```diff
@@ -6,68 +6,79 @@
    "source": [
     "# Quickstart\n",
     "\n",
-    "This notebook shows how to use PySTAC to read through the public Sentinel catalog and write a local version."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Reading STAC"
+    "This notebook is a quick introduction to using PySTAC for reading an existing STAC catalog. For more in-depth examples check out the [other tutorials](./tutorials).\n",
+    "\n",
+    "## Dependencies\n",
+    "\n",
+    "- PySTAC"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First, we want to hook into PySTAC to allow for reading of HTTP STAC items, as described in [the STAC_IO Concepts docs](concepts.html#using-stac-io). \n",
+    "## Reading a Catalog\n",
+    "\n",
+    "[A STAC Catalog](https://github.com/radiantearth/stac-spec/tree/master/catalog-spec) is used to group other STAC objects like Items, Collections, or even other Catalogs.\n",
+    "\n",
+    "We will be using a small example catalog adapted from the `geotrellis` [example Landsat Collection](https://github.com/geotrellis/geotrellis-server/tree/977bad7a64c409341479c281c8c72222008861fd/stac-example/catalog/landsat-stac-collection). All STAC Items and Collections can be found in the [docs/example-catalog](./example-catalog) directory of this repo; all Assets are hosted in the Landsat S3 bucket.\n",
```

Comment:
Should capitalize project name `GeoTrellis`. Also perhaps link to project homepage (https://geotrellis.io/)